ui/src/Components/Dashboard/Table/TableHeader/SortCursor.js

```diff
@@ -0,0 +1,39 @@
+import { useState } from "react";
+
+import { TABLE } from "../../../../Constant/constant";
+
+export default function SortCursor(props) {
+  const [currentClickedCursor, setCurrentClickedCursor] = useState("");
+
+  const onClickUpArrow = (args, headerName) => {
+    setCurrentClickedCursor(TABLE.UPCURSOR);
+    props.handleSorting(TABLE.ASCENDING, headerName);
+  };
+  const onClickDownArrow = (args, headerName) => {
+    setCurrentClickedCursor(TABLE.DOWNCURSOR);
+    props.handleSorting(TABLE.DESCENDING, headerName);
+  };
+
+  return (
+    <div>
+      <div
+        className={`up-arrow${
+          currentClickedCursor === TABLE.UPCURSOR ? '-clicked' : ''
```

Comment:
Can't we move `Up-arrow` and `clicked` to constant?

Reply:
Made some changes.For now I wont be reusing these classes anywhere else should we still move it constants?

Just curios! Whats your thought process when you are moving stuff to constants?

Reply:
I prefer to move everything to constant, tbh. As I avoid doing any hardcoding in the components